packages/core-components/src/layout/ProxiedSignInPage/ProxiedSignInPage.tsx

```diff
@@ -82,7 +82,11 @@ export const ProxiedSignInPage = (props: ProxiedSignInPageProps) => {
   if (status === 'loading') {
     return <Progress />;
   } else if (error) {
-    return <ErrorPanel title={t('proxiedSignInPage.title')} error={error} />;
+    return props.ErrorElement ? (
+      React.cloneElement(props.ErrorElement, { error })
```

Comment:
Uh, should prolly provide a component instead? :sweat_smile:, this isn't a pattern we use. Any particular upsides to this approach?

Reply:
The idea is that you can pass the error to the element as a prop. Is there another way? 😅

Reply:
one way could be to do something like:

```tsx
return props.renderError ? (
      props.renderError({ error })
    ) : (
      <ErrorPanel title={t('proxiedSignInPage.title')} error={error} />
    );
```

with `renderError` defined as: `renderError?(options: { error: Error }): React.ReactElement;`

Reply:
Ah, `render*` isn't really a good pattern anymore as it doesn't allow usage of hooks. We've used `components: { *: ComponentType }` in the scaffolder which mirrors MUI, but they're moving over to `slots: { ... }` in v7. I'm a bit worried adopting that straight away though, we'd need to be careful to have it behave the same as in MUI if we do. Thinking that just `ErrorComponent: ComponentType` as a prop might be the way to go

Reply:
Ah, true, that is the best option. I will refactor!